build.sbt

```diff
@@ -1,7 +1,7 @@
 import java.util.Properties
 
 val json4sVersion = "3.6.6"
-val circeVersion = "0.12.3"
+val circeVersion = "0.12.0-M3"
```

Comment:
What's the motivation for this change?

Reply:
Hi Tim
Yes, reverted to the original  `circeVersion`.